### PR TITLE
 gp-saml-gui: init at version 0.1 

### PR DESCRIPTION
**Pull request by pallix**
###### Description of changes

Add [gp-saml-gui](https://github.com/lkrms/gp-saml-gui) to the Nix tools.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/gp-saml-gui/default.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  src = fetchFromGitHub {
+    owner = "dlenski";
+    repo = "gp-saml-gui";
+    rev = "f1fafba";
+    sha256 = "sha256-7gIHShtpRhuYeADGz5EdhUcmnI7nUgtaHFYRS4uFvHI=";
+  };
+
+  buildInputs = with pkgs;
+    pkgs.lib.optional pkgs.stdenv.isLinux glib-networking;
```
Comment (turion) [inline pkgs/tools/networking/gp-saml-gui/default.nix:23]:
```suggestion
    lib.optional stdenv.isLinux glib-networking;
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by K900** [comment]
@ofborg build gp-saml-gui gp-saml-gui.passthru.tests

**Comment by pallix** [comment]
Added a commit / rebase to have the latest version.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add NAME`` into this PR.

**Comment by pallix** [comment]
> Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message `maintainers: add NAME` into this PR.

Thanks for taking time to review this. I have done within the latest rebase.

**Comment by pallix** [comment]
@SuperSandro2000 can it be merged ?

**Comment by pallix** [comment]
Thanks for the help!

**Pull request merged**